src/shogun/converter/Isomap.h

```diff
@@ -39,6 +49,9 @@ class CDistance;
  *
  * To use this converter with static interfaces please refer it by
  * sg('create_converter','isomap',k);
+
+ * cf. http://tapkee.lisitsyn.me/
+ * cf. https://en.wikipedia.org/wiki/Isomap
```

Comment:
A reference to the paper where Isomap was first introduced is a must as well :-)

http://isomap.stanford.edu/